analysis/monkmistweaver/src/modules/spells/EssenceFont.tsx

```diff
@@ -76,7 +93,25 @@ class EssenceFont extends Analyzer {
     };
   }
 
+  get suggestionThresholdsCancel() {
+    return {
+      actual: this.cancelled_ef,
+      isGreaterThan: {
+        major: 0,
+      },
+      style: ThresholdStyle.NUMBER,
+    };
+  }
   castEssenceFont(event: CastEvent) {
+    let extra_secs = 0;
+    if (this.hasUpwelling) {
+      extra_secs = Math.max(
+        (event.timestamp - (this.last_ef + SPELLS.ESSENCE_FONT.cooldown * 1000)) / 6000,
```

Comment:
Does the spell files actually have cooldown?
This is referring to 
common/spell/monk.ts

Reply:
Ah no it doesn't, think I confused Abilities.tsx in mw folder with the other monk file